docs/general/cloud-events.md

```diff
@@ -317,3 +317,20 @@ See [Customer Data](./customer-data).
     "applicationId": "9d311e9d-b8ff-491a-b558-cbe7293ee936"
 }
 ```
+
+## Customer Data Changed Event
```

Comment:
It is not only about `Customer`s but any type of `Subscription`. In https://my.tapio.one we call a `Subscription` `company account`. So I would use that name or simply `Subscription`.